### ISSUE TITLE
Missing AWS templates

### ISSUE THREAD

**Issue by Richard210363** [aws/aws-sdk-net#1451]
The templates for creating .Net applications that connect to AWS as opposed to those that run in AWS are no longer available.

## Expected Behavior
Based on the many tutorials from Amazon I was expecting to find templates such as "AWS Empty Project"

## Current Behaviour
This template is not available in The New Project dialog.
This means that tutorials such as https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CodeSamples.DotNet.html no longer work.

## Possible Solution
Add the missing templates back into the extension.

## Steps to Reproduce (for bugs)
Install AWS Toolkit for Visual Studio 2017.
Select File | New | Project.
Try to find any template that allows for .Net to connect and use AWS assets.

## Context
We are just starting to port our legacy finance system to AWS.  This will stop us.

## Your Environment
Platform.  .Net framework 4.72

**Comment by klaytaybai** [comment]
I'll investigate why these were removed and see if I can at least get permissions to release any old example templates you want.

**Comment by Richard210363** [comment]
Ta

**Comment by normj** [comment]
When we added VS 2019 support, which uses the same code as VS 2017 version, the new project wizard for extensions was pretty broken. So we removed the templates with the plan on creating a new set of templates shortly afterwards. Unfortunately like what happens too often in software the ramp up to re:Invent has postponed the work to getting a new set of templates out. It still high on my list to get the template situation fixed up soon and get the docs updated.

Out of curiosity can I ask you what types of templates you would like in VS? We definitely want the very basic console application like the previous AWS Empty Project and a web project. But I would appreciate any feedback of the types of templates that could be useful.
